### DISCUSSION TITLE
compatibility with UpgradeableSpawners

### DISCUSSION THREAD

**Discussion by KaBoom13** [BG-Software-LLC/SuperiorSkyblock2#1460] [Denied Feature Requests] [answered]
writes about adding compatibility with the 'Upgradeable Spawners' plugin to respawn mobs if there are more than one then it does not remove their limit

**Comment by OmerBenGera** [comment, marked as answer]
They should add support for SSB letting SSB know about the spawners being broken by either calling the BlockBreakEvent or by using Island#handleBlockBreak from SSB's API.
